README.rst

```diff
@@ -0,0 +1,47 @@
+==================================
+Spilo: HA PostgreSQL Clusters with Docker
+==================================
+
+Spilo is a Docker image that provides PostgreSQL and `Patroni <https://github.com/zalando/patroni>`__ bundled together. Patroni is a template for PostgreSQL HA. Multiple Spilos can create a resilient High Available PostgreSQL cluster. For this, you'll need to start all participating Spilos with `etcd <https://github.com/coreos/etcd>`__ and cluster name parameters. Spilo's name derives from სპილო [spiːlɒ], the Georgian word for "elephant."  
+
+Real-World Usage and Plans
+--------------------------
+
+Spilo is currently evolving: Its creators are working on a Postgres operator that would make it simpler to deploy scalable Postgres clusters in a Kubernetes environment, and also do maintenance tasks. Spilo would serve as an essential building block for this. There is already a `Helm chart <https://github.com/kubernetes/charts/tree/master/incubator/patroni>`__ that relies on Spilo and Patroni to provision a five-node PostgreSQL HA cluster in a Kubernetes+Google Compute Engine environment. (The Helm chart deploys Spilo Docker images, not just "bare" Patroni.)
+
+How to Use This Docker Image
+============================
+
+Spilo requires a load balancer (HAProxy, ELB, Google load balancer) to point to the master node. It provides two ways to achieve this: either by using the API URLs, or via the callback scripts. See Patroni for more details.
+
+Connecting to PostgreSQL
+------------------------
+**Administrative Connections**
+
+PostgreSQL is configured by default to listen to port 5432. The default Postgres user and database are created in the entrypoint with initdb.
```

Comment:
The reason it doesn't add any value is because it is quite vague: it refers to the Docker image entry point, which is unnecessary implementation detail, and mentions that default Postgres users and database is created with initdb, and this is *always* the case, no matter with Spilo or don't. Therefore, I think we should change it to 

"Spilo master initializes PostgreSQL and creates the superuser and replication user (`postgres` and `standby` by default)."